expyfun/tests/test_version.py

```diff
@@ -25,21 +25,18 @@ def test_version():
     assert_true(all('actual' in str(ww.message) for ww in w))
 
     v = 'b798e65'
```

Comment:
If you change this to a commit hash that happens to start with 0, it reveals a bug on [this line](https://github.com/LABSN/expyfun/pull/303/files#diff-2175524ba23bd6005be929c15c98478eR86)... need to remove the `[0]` in the LHS.  Noticed this by switching `v` here to the commit 090948e that led to #271